schema.json

```
@@ -159,7 +159,12 @@
         },
         "comment": {
           "description": "New comment on the column",
-          "type": "string"
+          "type": "string",
+          "goJSONSchema": {
+            "imports": ["github.com/oapi-codegen/nullable"],
+            "nillable": true,
+            "type": "nullable.Nullable[string]"
+          }
```

Ideally it would be something like this:

```suggestion
          "type": ["string", "null"],
```

---

The problem with using `type: ["string", "null"]` is that it generates the Go type:

```go
Comment *string `json:"name"
```

which doesn't allow us to distinguish between the case where the  'alter column' `comment` field is specified and set to `null`, vs left unspecified.

See https://github.com/xataio/pgroll/pull/345#issuecomment-2071930676

Using `nullable.Nullable` allows us to distinguish the 'set tno null' and the 'unspecified' cases.

---

Fair point, but from the user land perspective (instead of go code) I would like to see that the field can be set to null. So I think both things should be set, right?